jsapp/js/components/projectDownloads/projectExportsCreator.es6

```diff
@@ -428,23 +458,28 @@ export default class ProjectExportsCreator extends React.Component {
       </span>
     );
 
-    const customSelectionLabel = (
-      <span className='project-downloads__title'>
-        {t('Custom selection export')}
-      </span>
-    );
+    // make sure the order is right, can't trust object definition ordering :)
+    const exportMultipleOptions = [
+      EXPORT_MULTIPLE_OPTIONS.details,
+      EXPORT_MULTIPLE_OPTIONS.summary,
+      EXPORT_MULTIPLE_OPTIONS.both,
+    ];
 
     return (
       <div className='project-downloads__advanced-view'>
         <div className='project-downloads__column project-downloads__column--left'>
           <label className='project-downloads__column-row'>
             <span className='project-downloads__title'>
-              {t('Export select_multiple responses')}
+              {t('Export')}
+              &nbsp;
+              <em>{t('Select Many')}</em>
+              &nbsp;
+              {t('questions as…')}
```

Comment:
@magicznyleszek, this is a good example of a translation that will be difficult without context, especially for RTL languages. Maybe leaving the emphasis tags in the translatable string would be better(?)